monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -1640,8 +1645,9 @@ object Task extends TaskInstancesLevel1 {
     *
     *  It's a simple version of [[traverse]].
     */
-  def sequence[A, M[X] <: TraversableOnce[X]](in: M[Task[A]])
-    (implicit cbf: CanBuildFrom[M[Task[A]], A, M[A]]): Task[M[A]] =
+  def sequence[A, M[X] <: TraversableOnce[X]](
+    in: M[Task[A]]
+  )(implicit cbf: CanBuildFrom[M[Task[A]], A, M[A]]): Task[M[A]] =
```

Comment:
I don't like this one 😑 

Are the styles available documented somewhere?

Reply:
Yeah... unfortunately this is done by it's attempt to break up lines longer than 120 (as configured). You can usually get different results by manually putting certain curried parameter lists on separate lines. But automatically it always gives you these results... I don't think there is a way to configure which parameter lists ought to be moved to a newline, but I'll double check

The only documentation is here:
http://scalameta.org/scalafmt/#Configuration

From there onward it's trial and error. And I try to stay away from the list at the end, where certain properties can be removed without notice in future versions.